iac-extensions/terraform/src/test/java/org/sonar/iac/terraform/TestTreeBuilders.java

```diff
@@ -57,7 +58,7 @@ private SyntaxTokenBuilder() {
     }
 
     public static SyntaxToken token(String value) {
-      return new SyntaxTokenImpl(value, null, Collections.emptyList());
+      return new SyntaxTokenImpl(value, range(1, 0, value), Collections.emptyList());
```

Comment:
Not sure why we replaced the null range here?

Reply:
It was in an earlier attempt on this PR, where something was relying on this range. As I removed that code, I'll rollback this change as well.